mappings/net/minecraft/item/map/MapState.mapping

```diff
@@ -60,9 +117,27 @@ CLASS net/minecraft/class_22 net/minecraft/item/map/MapState
 		FIELD field_128 startZ I
 		FIELD field_129 startX I
 		FIELD field_130 dirty Z
+		FIELD field_27891 iconsDirty Z
 		METHOD <init> (Lnet/minecraft/class_22;Lnet/minecraft/class_1657;)V
 			ARG 1 player
 		METHOD method_111 markDirty (II)V
-			ARG 1 x
-			ARG 2 z
+			ARG 1 startX
+			ARG 2 startZ
 		METHOD method_112 getPacket (I)Lnet/minecraft/class_2596;
+			ARG 1 mapId
+		METHOD method_32375 getMapUpdateData ()Lnet/minecraft/class_22$class_5637;
+		METHOD method_32379 markIconsDirty ()V
+	CLASS class_5637 MapStateUpdateData
```

Comment:
Can this be named just `UpdateData` so it can be used as `MapState.UpdateData`?

Reply:
Good point.

With Enigma not remapping correctly subclasses it confused me a bit atm.

Reply:
Yeah, when navigating enigma navigates to nested classes than their outer instances. In addition cfr decompiler does not have nested class names recognized as tokens in source